src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -115,8 +118,26 @@ const SavedSearchAlertEditForm: React.FC<SavedSearchAlertEditFormProps> = ({
   onDeleteClick,
   onCompleted,
 }) => {
-  const { state, goToFilters, dispatch, onComplete } = useAlertContext()
+  const {
+    state,
+    goToFilters,
+    dispatch,
+    onComplete,
+    createAlertError,
+  } = useAlertContext()
   const { jumpTo } = useJump()
+  const { sendToast } = useToasts()
+
+  const mWebView = window.innerWidth < 1280
```

Comment:
Could we use the [env var](https://github.com/artsy/force/blob/009c1383869bc15157170523d385196e1e0153a0/src/Apps/Settings/Routes/SavedSearchAlerts/SavedSearchAlertsApp.tsx#L65) or [this](https://github.com/artsy/force/blob/009c1383869bc15157170523d385196e1e0153a0/src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSidebar/MyCollectionArtworkSidebarMetadata.tsx#L104) here?

Reply:
I tried both
env var always returns false
`const isMobile = __internal__useMatchMedia(THEME.mediaQueries.xs)` works, but isn't it only for internal use?

Reply:
Yeah, I guess, it's better to use `__internal__useMatchMedia`. `window` might be problematic when the page is rendered on the server ([SSR](https://www.google.com/search?q=ssr+window+&sca_esv=5a58adcc29de473b&sca_upv=1&rlz=1C5GCCM_en&sxsrf=ADLYWIL5tA15gS1rdcuNH5a4vpdGQz3mJw%3A1718359822270&ei=DhdsZqLxD5WX9u8P8tW0yAo&ved=0ahUKEwiisLWW7dqGAxWVi_0HHfIqDakQ4dUDCBE&uact=5&oq=ssr+window+&gs_lp=Egxnd3Mtd2l6LXNlcnAiC3NzciB3aW5kb3cgMgoQIxiABBgnGIoFMgUQABiABDIFEAAYgAQyBhAAGBYYHjIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHjIGEAAYFhgeMgYQABgWGB4yCBAAGBYYHhgPSKk7UPEIWJMLcAN4AJABAJgBXqAB1gKqAQE0uAEDyAEA-AEBmAIGoAKnAsICChAAGLADGNYEGEfCAg0QABiABBiwAxhDGIoFwgILEAAYgAQYhgMYigWYAwCIBgGQBgmSBwE2oAeFHw&sclient=gws-wiz-serp)). It's okay to use it in an effect or in callbacks, but it might be undefined if the component isn't rendered in the browser.